style.css

```diff
@@ -0,0 +1,136 @@
+
+.basic-section {
+    width: 100%;
+    height: 100vh;
+    background-image: url("img/cover.jpg");
+    background-position: center;
+    background-size: cover;
+    opacity: 0.9;
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(48, 49, 70); 
+    /*margin: 70px;*/
+    margin-top: 0;
+    color: white;
+    max-width: 1440px;
+}
+
+
+.nav{
+    display: flex;
+}
+
+.nav_bar{
+    display: flex;
+    margin: 450px;
+    margin-top: 40px;
+    list-style: none;
+    font-weight: bold;
+    font-size: 14px;
+    color: white;
+}
+.nav_bar a{
+    padding: 15px;
+    text-decoration: none;
+    color: white;
+    transition: 0.2s
+}
+
+.nav_bar li{
+        margin: 5px;
+}
+
+.nav_bar a:hover{
+    color: rgb(43, 27, 116);
+}
+
+.logo{
+    margin: 30px;
+    margin-top: 0;
+    font-size: 24px;
+}
+
+
+.main_content{
+    text-align: center;
+    margin-left: 30px;
+    margin-top: -280px;
+    
+
+}
+
+.maintext{
+    text-align: center;
+    font-weight: 700;
+    font-size:58px ;
+    margin: 0;
+}
+
+.maintext2{
+    text-align: center;
+    font-size: 20px;
+    font-weight: 500;
+}
+.buttonquote, .buttonquote2{
+    color: antiquewhite;
+    font-size: 14px;
+    text-decoration: none;
+    font-weight: 700;
+    border: 2px solid;
+    padding: 10px 25px 10px 25px;
+    border-radius: 37px;
+    /* padding-left: 25px; Отступ слева
+    padding-right: 25px; */
+    margin: -10px;
+    transition: 0.2s;
+}
+
+.buttonquote{
+    color: antiquewhite;
```

Comment:
don't write same styles again. You already wrote them in complex selector for both buttons. 
And I really suggest you to add one more common class "button" to both and add all common styles into common class.